apps/core/scenes/passengerForm/BaggageBundles/BaggageBundle.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+
+import React from 'react';
+import { View } from 'react-native';
+import { StyleSheet, RadioButton } from '@kiwicom/universal-components';
+import { Text } from '@kiwicom/margarita-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import BaggageTypeDetail from './BaggageTypeDetail';
+
+type Props = {|
+  +price: ?string,
+  +bags: any,
+  +bagIndex: number,
+  +onToggleCheck: any,
+  +type: string,
+  +currentlyChosen: number,
+|};
+
+type State = {|
+  checked: boolean,
+|};
+
+export default class BuggageBundle extends React.Component<Props, State> {
+  state = {
+    checked: this.props.bagIndex === 0,
+  };
+
+  handlePress = () => {
+    const { type, bagIndex } = this.props;
+    this.setState(state => {
+      return {
+        ...state,
+        checked: !state.checked,
+      };
+    }, this.props.onToggleCheck(type, bagIndex));
+  };
+
+  render() {
+    const { currentlyChosen, bagIndex, bags, price } = this.props;
+    const shouldBeChecked = currentlyChosen === bagIndex;
+
+    return (
+      <View
+        style={[styles.container, shouldBeChecked && styles.containerChecked]}
+      >
+        <View style={styles.row}>
+          <RadioButton
+            type="check"
+            checked={shouldBeChecked}
+            onPress={this.handlePress}
+          />
+          <View style={styles.baggageDetails}>
+            {bags.map((bag, index) => {
+              return (
+                <BaggageTypeDetail
+                  key={index}
+                  bagType={bag.type}
+                  quantity={bag.quantity}
+                  dimensions={bag.dimensions}
+                  weight={bag.weight}
+                  chosen={shouldBeChecked}
+                />
+              );
+            })}
+          </View>
+        </View>
+        <Text>{price}</Text>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    padding: parseFloat(defaultTokens.spaceXSmall),
+    marginTop: parseFloat(defaultTokens.spaceSmall),
+    marginHorizontal: parseFloat(defaultTokens.spaceSmall),
+    marginBottom: parseFloat(defaultTokens.spaceXXXSmall),
+    borderWidth: 0.5,
+    borderStyle: 'solid',
+    borderColor: defaultTokens.borderColorTableCell,
+    borderRadius: 4,
+    shadowColor: defaultTokens.paletteInkDark,
+    shadowOffset: {
+      width: 0,
+      height: 2,
+    },
+    shadowOpacity: 0.3,
+    shadowRadius: 4,
+  },
+  row: {
+    flexDirection: 'row',
+  },
+  containerChecked: {
+    borderColor: defaultTokens.borderColorButtonPrimaryBordered,
+    borderWidth: 2,
+    marginTop: parseFloat(defaultTokens.spaceSmall) - 1.5,
```

Comment:
Why there is `-1.5`?

Reply:
Because component's border width was 0.5 before and then the component's state become checked, the border width is 2. So I am changing margin in the checked state, so the component wouldnt be moving on toggle

Reply:
I understand now. And what about define some constants and use them?

Something like (but with better naming):
```js
const borderWidth = 0.5
const checkedBorderWidth = 2
...
marginTop: parseFloat(defaultTokens.spaceSmall) - checkedBorderWidth - borderWidth,

```

Reply:
Yes, much better! there wont be confusion anymore